loadbalancer/conf/loadbalancer.nginx.conf

```diff
@@ -0,0 +1,55 @@
+# Setup logging to include request time and bytes sent, 
+log_format pkgserver_logformat '$remote_addr [$time_local] "$request" $status $body_bytes_sent "$http_user_agent" $request_time $http_julia_version $http_julia_system "$http_julia_ci_variables" $http_julia_interactive "$http_julia_pkg_server"';
+
+# This upstream uses `$request_uri` hashing to ensure that the same resource request goes to the same server
+# this allows us to essentially shard the space of resources across servers
+upstream pkgservers_hashed {
+    hash $request_uri consistent;
+${PKGSERVERS_SERVER_BLOCK}
+}
+
+# This upstream uses round-robin request handling
+upstream pkgservers_roundrobin {
+${PKGSERVERS_SERVER_BLOCK}
+}
+
+server {
+    listen              443 ssl http2;
+    listen              [::]:443 ssl http2;
+    server_name         ${REGION}.pkg.julialang.org loadbalancer-${REGION}.ip.cflo.at loadbalancer-${REGION}.ipv4.cflo.at loadbalancer-${REGION}.ipv6.cflo.at;
+    ssl_certificate     /etc/letsencrypt/live/${FQDN}/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/${FQDN}/privkey.pem;
+    
+
+    # In general, let's ask a different upstream if the one we're talking to errors out.
+    proxy_next_upstream error timeout;
+
+    # We pass all content-addressed URLs off to consistent PkgServer locations,
+    # so that each package server only needs to keep track of a portion of these resources
+    location ~ /(registry|package|artifact|mirror)/ {
+        set $upstream pkgservers_hashed;
+        try_files $uri @loadbalance;
```

Comment:
These will always hit the fallback, right?

Reply:
Yes, they always go to the `@loadbalance` location.